test/integrations/destinations/google_adwords_remarketing_lists/router/data.ts

```diff
@@ -228,7 +228,14 @@ export const data = [
                     'Content-Type': 'application/json',
                     'developer-token': 'ijkl9101',
                   },
-                  params: { listId: '7090784486', customerId: '7693729833', consent: {} },
+                  params: {
```

Comment:
We need to add new test cases for the scenarios where consent data is specified.